compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -983,7 +984,16 @@ TR_Debug::print(TR::FILE *pOutFile, TR::MemoryReference *mr)
 void
 TR_Debug::printARM64GCRegisterMap(TR::FILE *pOutFile, TR::GCRegisterMap *map)
    {
-   TR_UNIMPLEMENTED();
+   TR::Machine *machine = _cg->machine();
+
+   trfprintf(pOutFile, "    registers: {");
+   for (int i = 31; i >= 0; i--)
```

Comment:
While this is still informationally correct, is there a particular reason you count downwards in this loop?  Presumably because this was lifted from the big endian Power implementation?  All things being the same, I would prefer the loop to count up so that the lowest register numbers appear first.

Reply:
I have no particular reason for that.  Just because both p and arm32 versions of this function counts down.
Changed the code to count up.